src/modules/build.coffee

```diff
@@ -153,6 +153,7 @@ module.exports = (Smasher) ->
       logConnections: args.verbose
       logFileChanges: args.verbose
       # logLevel:     'debug'
+      notify:         false
```

Comment:
There it is! I knew there had to be an option somewhere. I'll probably make it user-configurable at some point, but I've got no problem defaulting to this for now.